authenticationToken/authenticationSample.js

```diff
@@ -0,0 +1,70 @@
+
+
+function getAuthToken() {
+    var enterpriseip = $('#enterpriseip').val();
+    var miccAuthServerPath = `http://${enterpriseip}/authorizationserver`
+    var username = $('#username').val();
+    var password = $('#password').val();
+    var data = `grant_type=password&username=${username}&password=${password}`;
+    var bearerToken = "";
+
+    $.ajax({
+        type: "POST",
+        url: `${miccAuthServerPath}/token`,
+        data: data,
+        headers: {
+            "Content-Type": "application/x-www-form-urlencoded"
+        }
+    }).done(function (data) {
+            console.log('Login success.  Received data:  ', data);
+            bearerToken = data.access_token; 
+            $('#authToken').text(bearerToken);
+            $('#authTokenCard').removeClass('invisible');
+
+    });
+}
+
+function getAuthTokenClaims() {
+    //request claims from api
+    var enterpriseip = $('#enterpriseip').val();
+    var apiPath = `http://${enterpriseip}/authorizationserver/api/account`;
+
+        $.ajax({
+            method: 'GET',
+            url:apiPath,
+            beforeSend: function (xhr) {
+                console.log('Bearer ' + $('#authToken').html());
+                xhr.setRequestHeader('Authorization', 'Bearer ' + $('#authToken').html());
+            }
+        }).done(function (result) {
+            console.log('Claims ' + JSON.stringify(result));
+            //$('#result').html(JSON.stringify(result));
+            $.each(result, function(i, item){
+                var $tr = $('<tr>').append(
+                    $('<td>').text(i),
+                    $('<td>').text(item),
+                    $('</tr>')
+                ); 
+                $('#claimsTableBody').append($tr);
+                $('#claimsTable').css({'visibility': 'visible'});   
+            });
+        });
+}
+
+
+function test(){
+    $.ajax({
+        type: 'GET',
+        url: 'https://cgoodyear.mitel.com/miccsdk/api/v1/openmedia',
```

Comment:
I would remove your machine name from this. probably just replace with localhost or the $enterpriseip var